src/Sylius/Behat/Context/Api/Admin/ManagingOrdersContext.php

```diff
@@ -311,4 +315,34 @@ public function theAdministratorShouldSeeThatThisOrderHasBeenPlacedIn(
 
         Assert::same($currencyCode, $currency);
     }
+
+    /**
+     * @Then I should see an order with :orderNumber number
+     */
+    public function iShouldSeeOrderWithNumber($orderNumber)
```

Comment:
```suggestion
    public function iShouldSeeOrderWithNumber(string $orderNumber): void
```

and the same in other methods